runtime/src/bank.rs

```diff
@@ -252,6 +252,14 @@ impl AddAssign for SquashTiming {
     }
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, Default, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct CollectorFeeDetails {
+    pub transaction_fee: u64,
+    pub priority_fee: u64,
+    pub reserved: u64,
```

Comment:
does this have a known purpose? Will it be enough? Will it ever be used?

Reply:
it is reserved for potential write-lock fee ([SIMD-0110](https://github.com/solana-foundation/solana-improvement-documents/pull/110)), which would be 100% burnt. It is not yet approved, but there is a chance it will, want to be greedy to have space allocated for it for now 😸